src/content/contributing/release-process/_index.en.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Release process"
+date: 2020-03-18T16:03:26+01:00
+weight: 10
+---
+
+
+In this section of the documentation we explain the necessary steps to make a submariner release,
+it's assumed that you are familiar with the submariner project and the different repositories.
+
+
+# Step 1: release submariner (core)
+
+
+Assuming that you have an existing submariner git directory, and that you are making a release based in master.
+
+```bash
+cd submariner
+git remote add upstream ssh://git@github.com/submariner-io/submariner
+git fetch -a -v -t
+git checkout remotes/upstream/master -B master
+git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
+git push upstream v0.2.0-rc0
+```
+
+At this point the release should appear here, a github release can be created.
+
+> https://github.com/submariner-io/submariner/tags
+
+A build should start:
+
+> https://travis-ci.com/github/submariner-io/submariner/branches
+
+Look under the "Active branches" section for v0.2.0-rc0 , and monitor that the build
+completes successfully, specially the Deploy section, where the images are pushed to quay.io, those should eventually appear here after the job has finished:
+
+For this example the build can be found [here](https://travis-ci.com/github/submariner-io/submariner/builds/153943761) from all the sub-builds, the one tagged with DEPLOY=true will push the resulting image to quay, as can be seen here: [deployment script](https://travis-ci.com/github/submariner-io/submariner/jobs/299505392#L3417)
```

Comment:
I'm not sure this detail adds any value and may be confusing. Also wouldn't the specific links eventually become stale/non-existent?

Reply:
Yeah, I also thought of that. But it'd keep it for now. Some people need to "touch and see" stuff closely to understand better, but it's likely the links will expire, sadly.

Reply:
Not sure if you are trying, but you are being   ;-) (:heart:) We can always replace that later with a better version or screenshots or enhance it, but we need to deliver something to start with.

Reply:
Well I'm trying to tell you that as a first-time reader w/o having performed a release that this section, at least as presented, is a bit confusing and doesn't make sense. It makes sense to you b/c you know how everything works and have done it many times. I'd prefer to just remove it and possibly add an enhanced version later if needed based on feedback rather than leave it in and possibly confuse readers. But if you don't want to take my feedback on this then, fine, leave it.

Reply:
I will switch that paragraph for screenshots on a second iteration, let me open a bug:

https://github.com/submariner-io/submariner-website/issues/46

I promise I will handle it, but I want to deliver something for 0.2.0